tests/chainer_tests/test_variable.py

```diff
@@ -507,6 +507,17 @@ def test_unchain(self):
         assert ret[1].rank == old_rank
         self.check_backward((ret[1],), (ret[2],), (ret[3],), False)
 
+    def test_unchain_split(self):
+        if not len(self.x.shape) > 0:
+            return
+        ret = get_variable(np, self.x)
```

Comment:
It looks ok to use `x` for the variable (without the confusion with `self.x`).

Reply:
`x` is the class variable.